packages/arb-ts/src/lib/inbox/inbox.ts

```diff
@@ -183,6 +184,38 @@ export class InboxTools {
       }
     )
 
+    if (events.length !== 0) return events
+    else if (maxedSearchRangeBlocks === maxSearchRangeBlocks) return []
+    else {
+      return await this.getEventsAndIncreaseRange(
+        bridge,
+        searchRangeBlocks * 2,
```

Comment:
we could make the `* 2` a default parameter like "growthRate" that defaults to 2
```suggestion
        searchRangeBlocks * 2,
```